src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
Return [Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html) when you can return `null` in DAO.

```suggestion
    Optional<Manufacturer> get(Long id);
```